From 8125b0a3d1a167dcaca9f62bd60c7f8b9d1f7e35 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Caol=C3=A1n=20McNamara?= <caolanm@redhat.com>
Date: Fri, 5 Feb 2016 14:03:44 +0000
Subject: [PATCH] Resolves: tdf#97465 like wheel ignore swipe for sc input
 handler inputchanged

(cherry picked from commit e35f3b6a3357fc3832a9d68ed37ddb9b5320ef0a)
(cherry picked from commit 113f000d38c34dd4dda6903976f8febf6d95375c)

Change-Id: Ide7fe1388ffe6f85a1f459037316d03193470d8a
---
 sc/source/ui/app/inputwin.cxx | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/sc/source/ui/app/inputwin.cxx b/sc/source/ui/app/inputwin.cxx
index b3776fd..834f4a6 100644
--- a/sc/source/ui/app/inputwin.cxx
+++ b/sc/source/ui/app/inputwin.cxx
@@ -1646,6 +1646,10 @@ void ScTextWnd::Command( const CommandEvent& rCEvt )
         {
             //don't call InputChanged for CommandEventId::Wheel
         }
+        else if ( nCommand == CommandEventId::Swipe )
+        {
+            //don't call InputChanged for CommandEventId::Swipe
+        }
         else
             SC_MOD()->InputChanged( pEditView );
     }
-- 
2.5.0

